scripts/load_testing/server_config.py

```diff
@@ -0,0 +1,100 @@
+import argparse
+from math import ceil, floor, log
+import os
+
+INSTANCES = 3
+PREFIX = 'test'
+PELIKAN_ADMIN_PORT = 9900
+PELIKAN_SERVER_PORT = 12300
+PELIKAN_SERVER_IP = '10.25.2.45'
+PELIKAN_SIZE = 64
+PELIKAN_SLAB_MEM = 4294967296
+PELIKAN_BINARY = '/root/Twitter/pelikan/_build/_bin/pelikan_twemcache'
+THREAD_PER_SOCKET = 48
+BIND_TO_CORES = False
+BIND_TO_NODES = True
+
+def generate_config(prefix, instances, hash_power, slab_mem):
+  # create top-level folders under prefix
+  config_path = os.path.join(prefix, 'config')
+  os.makedirs(config_path)
+  log_path = os.path.join(prefix, 'log')
+  os.makedirs(log_path)
+
+  # create twemcache config file(s)
+  for i in range(instances):
+    admin_port = PELIKAN_ADMIN_PORT + i
+    server_port = PELIKAN_SERVER_PORT + i
+    config_file = 'pelikan-{server_port}.config'.format(server_port=server_port)
+    config_str = """\
+daemonize: yes
+admin_port: {admin_port}
+server_port: {server_port}
+
+buf_init_size: 4096
+
+buf_sock_poolsize: 16384
+
+debug_log_level: 5
+debug_log_file: log/{server_port}/twemcache.log
```

Comment:
Need to add prefix before log paths in config.

Reply:
Shall we assume we launch `bring-up.sh` from the test directory, e.g. `test_32_4`, instead of one level up? This makes the test folder self-contained, i.e. `bring-up.sh` assumes `pwd` is where itself resides, and all paths are relative to that.

This actually means we should _remove_ prefix from the `config_path` parameter below. And do something like the following to launch an experiment
```
cd pelikan_32_4 && bring-up.sh && cd ..
cd rpcperf_100_32_4 && run.sh && cd ..
```

Reply:
Sounds good.